octobot/src/server/github_handler.rs

```diff
@@ -11,7 +11,7 @@ use octobot_lib::config::Config;
 use octobot_lib::errors::Result;
 use octobot_lib::github;
 use octobot_lib::github::api::Session;
-use octobot_lib::github::CommentLike;
+use octobot_lib::github::{CommentLike, PullRequest, Repo};
```

Comment:
i've been trying to avoid pulling the model names directly into the namespace here since there's so much going on in this file. so like, `github::PullRequest`, `github::Repo`.

(the only reason to pull in CommentLike here is because it's a trait and rust needs it imported.